File: Resueltos_por_competidor/Juan_Martinez/MultiplicationTable.cpp
/*
 * Autor: EroMergeSort del TCC
 * Problema: Multiplication Table 
 * Juez online: Vjudge
 * Veredicto: Accepted
 * Url: https://vjudge.net/problem/CodeForces-577A
 */
#include <bits/stdc++.h>

using namespace std;

typedef long long ll;

int main() {

    ll a, b, sum, con = 0;
    cin >> a >> b;
    
    for(ll i = 1; i <= a; i++){
        if((i*a) >= b){
            if(b % i == 0) con++;
        }
    }
    
    cout<<con<<endl;
}

File: Resueltos_por_competidor/Gabriella_Castro/MultiplicationTable.java
/*
 * Autor: Gabriella Castro
 * Problema: Multiplication Table
 * Juez online: VJudge
 * Veredicto: Accepted
 * URL: https://vjudge.net/problem/CodeForces-577A
 */

import java.util.Scanner;

public class Main{

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        long n = sc.nextLong();
        long x = sc.nextLong();

        int cont = 0;

        for (long i = 1; i <= n && i <= x; i++) {

            if (x % i == 0) {
                long j = x / i;

                if (j <= n) {
                    cont++;
                }
            }
        }
        System.out.println(cont);
    }

}
